src/translations/nbTranslations.json

```diff
@@ -691,6 +690,8 @@
       "files_uploaded_other": "{{count}} filer lastet opp",
       "general_support_pending": "Venter på oppfølging",
       "get_curator_support": "Få hjelp av kurator",
+      "imported_from_source": "Importert fra {{source}}",
+      "imported_from_source_and_archive": "Importert fra {{source}} ({{archive}})",
```

Comment:
På tide med eget objekt for logg-tekst?

Reply:
godt poeng det egentlig. Loggen hører eeegentlig ikke bare inn under `my_page.messages`. Men ikke noe must å flytte (nå) for min del iaf

Reply:
Flytter når jeg redesigner loggen